pkg/keccak/Makefile

```
@@ -0,0 +1,24 @@
+PKG_NAME    := keccak
+PKG_URL     := https://github.com/gvanas/KeccakCodePackage
```

This URL redirects to https://github.com/XKCP/XKCP. Should be changed ?

---

I tested and https://github.com/XKCP/XKCP works the same. Sooo:
```suggestion
PKG_URL     := https://github.com/XKCP/XKCP
```

---

Will be in the next commit.

---

The URL is still the old one :/

---

Well, I haven't made the commit yet. ^^ I wanted to address all your suggestions at once.